docs/source/configuration-reference.rst

```diff
@@ -3,29 +3,98 @@ Configuration Reference
 
 When defining nested configurations, the top level default value will be **blank**, with subsequent keys and values listed below
 
+Dask
+----
 
-Scheduler AUTODOC
------------------
+.. dask_config_to_html::
+    :location: dask
+    :config: https://raw.githubusercontent.com/dask/dask/master/dask/dask.yaml
+    :schema: https://raw.githubusercontent.com/dask/dask/93356b0415f6c0c192e527d5be57f55e88b7b84a/dask/dask-schema.yaml
 
-.. autoconfigurable:: distributed.scheduler
-    https://raw.githubusercontent.com/dask/distributed/d027aeae839d5f502141ab7ca3e70ed516f07cf5/distributed/distributed.yaml
-    https://raw.githubusercontent.com/dask/distributed/d027aeae839d5f502141ab7ca3e70ed516f07cf5/distributed/distributed-schema.yaml
 
+Distributed
+-----------
 
-Scheduler HTML
---------------
+Client
+~~~~~~
 
-.. yamltohtml::
-    distributed.scheduler
-    https://raw.githubusercontent.com/dask/distributed/d027aeae839d5f502141ab7ca3e70ed516f07cf5/distributed/distributed.yaml
-    https://raw.githubusercontent.com/dask/distributed/d027aeae839d5f502141ab7ca3e70ed516f07cf5/distributed/distributed-schema.yaml
+.. dask_config_to_html::
+    :location: distributed.client
+    :config: https://raw.githubusercontent.com/dask/distributed/70700a1059fdae542ddbb3534f3caa3d27ca2e5d/distributed/distributed.yaml
+    :schema: https://raw.githubusercontent.com/dask/distributed/70700a1059fdae542ddbb3534f3caa3d27ca2e5d/distributed/distributed-schema.yaml
+
+Comm
+~~~~
+
+.. dask_config_to_html::
+    :location: distributed.comm
+    :config: https://raw.githubusercontent.com/dask/distributed/70700a1059fdae542ddbb3534f3caa3d27ca2e5d/distributed/distributed.yaml
+    :schema: https://raw.githubusercontent.com/dask/distributed/70700a1059fdae542ddbb3534f3caa3d27ca2e5d/distributed/distributed-schema.yaml
+
+
+Dashboard
+~~~~~~~~~
+
+.. dask_config_to_html::
+    :location: distributed.dashboard
+    :config: https://raw.githubusercontent.com/dask/distributed/70700a1059fdae542ddbb3534f3caa3d27ca2e5d/distributed/distributed.yaml
```

Comment:
I assume you intend to change this when https://github.com/dask/distributed/pull/3696 is merged, just making a note so it's not forgotten.

Reply:
yup yup, still futzing around with a few things